README.md

```diff
@@ -74,7 +74,29 @@ $version = XdebugHandler::getSkippedVersion();
 ```
 
 ### Output
-The library is silent unless the `-vvv` command-line option is present, which results in the output of simple status messages.
+The `setLogger` method enables the output of status messages to an external PSR3 logger.
+
+```php
+use Composer\XdebugHandler\XdebugHandler;
+
+$xdebug = new XdebugHandler('myapp');
+// Provide a PSR3 logger
+$xdebug->setLogger($myLogger);
+```
+
+All messages are prefixed `xdebug-handler:` and are reported with either `DEBUG` or `WARNING` log levels. No context data is provided. For example:
```

Comment:
What is the benefit of promising that no context is provided? Maybe we want to in the future and then it would be a breaking change

Reply:
Each message is self-contained. Should we ever need to include more information, we will provide another message.